testing/kuttl/e2e-other/streaming-standby/00--secrets.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kuttl-test-streaming-cluster
+---
+apiVersion: v1
+data:
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJnakNDQVNpZ0F3SUJBZ0lSQU0vVDF1MXllNHZ3ek1SWEt1NGlWZVF3Q2dZSUtvWkl6ajBFQXdNd0h6RWQKTUJzR0ExVUVBeE1VY0c5emRHZHlaWE10YjNCbGNtRjBiM0l0WTJFd0hoY05Nakl3TlRJek1UZ3hNRFE0V2hjTgpNekl3TlRJd01Ua3hNRFE0V2pBZk1SMHdHd1lEVlFRREV4UndiM04wWjNKbGN5MXZjR1Z5WVhSdmNpMWpZVEJaCk1CTUdCeXFHU000OUFnRUdDQ3FHU000OUF3RUhBMElBQlArM2dEb2s3V3duaDZmNnNUV3ozUmlrS3Q4TFhyN0QKSEpGSGNXdHd3MDI5TXQrb0lubWYwUE1VS1BiVHgrSDBSZTBLcENSRUhCbytmcXJqblIzZlBXdWpSVEJETUE0RwpBMVVkRHdFQi93UUVBd0lCQmpBU0JnTlZIUk1CQWY4RUNEQUdBUUgvQWdFQU1CMEdBMVVkRGdRV0JCU3ErUFdhClQreTAvUjBRb1AzUS9nUnZWY3JGQ0RBS0JnZ3Foa2pPUFFRREF3TklBREJGQWlBRHl3UXR2Zk1xUEIvWXlzL1QKd2lNZExNR3JocWVXeDVjYVZ2TWNVWkJxWHdJaEFQS1NBemo5K1RsTzg0cmNFN25pT3U2K2NRWEYzcjNxTFFOYQpNYWVId3d5TAotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
Probably not worth it to leave a note to update these certs before they expire in 2032, yeah?

Reply:
I would think not, hopefully by then we will be able to use something like cert manager